bootloaders/riotboot_dfu/main.c

```diff
@@ -27,6 +27,15 @@
 #include "riotboot/slot.h"
 #include "riotboot/usb_dfu.h"
 
+#ifdef ALTERNATIVE_MODE
+extern bool board_bootloader_alternative_mode(void);
+#else
+static bool bootloader_alternative_mode(void)
+{
+    return false;
```

Comment:
I like feature consistency across boards without much code duplication, so how about

```suggestion
    gpio_init(BTN_BOOTLOADER_PIN, BTN_BOOTLOADER_MODE);
    return gpio_read(BTN0_PIN) != BTN_BOOTLOADER_INVERTED;
```

with a

```C
#if !defined(BTN_BOOTLOADER_PIN) && defined(BTN0_PIN)
#define BTN_BOOTLOADER_PIN BTN0_PIN
#define BTN_BOOTLOADER_MODE BTN0_MODE
#define BTN_BOOTLOADER_INVERTED false
#endif
```

on top?